app/services/google/keyword_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Google
+  class KeywordService
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                 'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36'
+
+    BASE_SEARCH_URL = 'https://www.google.com/search'
+
+    def initialize(keyword, lang: 'en')
```

Comment:
```suggestion
    def initialize(keyword, lang: :en)
```

Whenever possible, using symbol is better for performance (and shorter to write ^^)